includes/class.llms.person.handler.php

```diff
@@ -964,7 +964,14 @@ public static function validate_fields( $data, $screen = 'registration' ) {
 			// Check email field for uniqueness.
 			if ( 'email_address' === $name ) {
 
-				$skip_email = false;
+				/**
+				 * Allow for skipping of the emxail_exists check.
+				 *
+				 * @since x.x.x
```

Comment:
Per doc standards: https://github.com/gocodebox/lifterlms/blob/trunk/docs/documentation-standards.md#using-placeholders

```suggestion
				 * @since [version]
```